server/templates/static/homepage.html

```diff
@@ -57,119 +57,207 @@ <h2>{% trans %}What's new{% endtrans %}</h2>
+        <li>Max Temperatures under RCP 4.5 in
+          <a class="sub" href="{{ url_for('tools.map') }}#sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&pd=country/IND&pn=India&pt=Country&ept=AdministrativeArea2">{{ place_names['country/IND'] }}</a> and
+          <a class="sub" href="{{ url_for('tools.map') }}#sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&pd=country/USA&pn=United%20States%20of%20America&pt=Country&ept=County">{{ place_names['country/USA'] }}</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&svy=Percent_Person_WithCoronaryHeartDisease&epd=country/USA&epn=United%20States%20of%20America&epts=Country&ept=County&qd=1&ld=0&dd=1&ct=0&rg=0">Prevalence Of Coronary Heart Disease vs Max Temperature under RCP 4.5 for USA Counties</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.timeline') }}#statsVar=DifferenceAcrossModels_Max_Temperature&place=geoId/0668000">Difference across models for {{ place_names['geoId/0668000'] }}</a>
+        </li>
+        <li>
+          <a class="sub" href="{{ url_for('tools.map') }}#&sv=NumberOfMonths_WetBulbTemperature_35COrMore_RCP45_MinRelativeHumidity&pc=0&denom=Count_Person&dt=2099&pd=country/IND&pn=India&pt=Country&ept=AdministrativeArea2">Districts in {{ place_names['country/IND'] }} with Wet Bulb Temp > 35°C</a>
+        </li>
+      </ul>
+    </li>
     <li>
-      {# TRANSLATORS: Header for a list of links to the Place Explorer product. Please see this string in action at https://datacommons.org #}
-      <a class="main" href="{{ url_for('place.place') }}">{% trans %}Explore Places{% endtrans %}</a>:
+      <span class="main">Emissions</span>:
       <ul>
         <li>
-          {% set geoid='geoId/0649670' %}
-          <a class="sub" href="{{ url_for('place.place', place_dcid=geoid) }}">{{ place_names[geoid] }}</a>,
+          <a class="sub" href="{{ url_for('tools.map') }}#&sv=Annual_Emissions_CarbonDioxide_NonBiogenic&pc=0&denom=Count_Person&pd=geoId/48&pn=Texas&pt=State&ept=County&ppt=EpaReportingFacility&mp=1">Carbon Dioxide Emissions from Large Facilities in Texas</a>
         </li>
         <li>
-          {% set geoid='geoId/3651000' %}
-          {# TRANSLATORS: Link to a Place page showing data related to Health for a place { cityName }. Please keep the string `%(cityName)s` as is in the final translation. Please see this string in action as "New York City Health" at https://datacommons.org #}
-          <a class="sub" href="{{ url_for('place.place', place_dcid=geoid, topic='Health') }}">{% trans cityName=place_names[geoid] %}{{ cityName }} Health{% endtrans %}</a>,
+          <a class="sub" href="{{ url_for('ranking.ranking', stat_var='Annual_Emissions_GreenhouseGas_NonBiogenic', place_type='State', place_dcid='country/USA') }}">Greenhouse gas emission ranking for US States</a>
```

Comment:
nit: "US states ranked by Greenhouse gas emissions" ... (feels like it reads slightly better)

Reply:
nit: "Greenhouse Gas Emissions"? to match the capitalization of "Carbon Dioxide Emissions" in the example before this one